Yoast/wordpress-seo
# Move the CornerstoneToggle to Wordpress-seo instead of Yoast-components

## Summary

This PR can be summarized in the following changelog entry:

* The CornerstoneToggle was moved from yoast-components to the Plugin.

## Relevant technical choices:

* The Toggle is only used in the plugin, therefore it seemed logical to move it to the plugin.

## Test instructions
This PR can be tested by following these steps:

* Check if the "new" CornerstoneToggle is used
* Check if everything still works (build succeeds, test run successful)

## Quality assurance


Fixes Yoast/javascript#37

@andizer will do the code review and acceptance test because he has the most context on the plugin and on the issue that was created.
